aristanetworks/ansible-avd
# Feat(eos_designs): Update platform.py with 7358X4

## Change Summary



## Related Issue(s)

Fixes #<ISSUE ID>

## Component(s) name

`arista.avd.eos_designs`

## Proposed changes
Set the default management interface for the 7358X4 series to `Management1/1`

## How to test
Set the platform type to `7358X4` - the management interface should render as `Management1/1`

## Checklist

### User Checklist


- N/A

### Repository Checklist
